### ISSUE TITLE
Tests on Mac can't toggle dialog through keyboard command

### ISSUE THREAD

**Issue by Plagiatus** [JirkaDellOro/FUDGE#408]
On the MacBookPro I am unable to click F1 while holding down the control key. It literally doesn't fire the keydown event while holding control.

**Comment by JirkaDellOro** [comment]
Only with the F-Keys or doesn't it fire with all keys while holding control?

**Comment by Plagiatus** [comment]
Seems to be only F keys, Arrow Keys, Capslock and Tab (and fn) that are affected. everything else is seemingly working.

**Comment by JirkaDellOro** [comment]
So Ctrl-Arrow won't work? That's bad...

**Comment by Plagiatus** [comment]
yeah, seems like those have special effects on mac, like Ctrl-Down opens up an overview of active programs.
